Utilities/Thread.cpp

```diff
@@ -1123,7 +1123,7 @@ bool handle_access_violation(u32 addr, bool is_writing, x64_context* context)
 			g_tls_fault_rsx++;
 			if (cpu && cpu->test_stopped())
 			{
-				std::terminate();
+				//
```

Comment:
You should probably leave an explanation on why the `if` block has no code in it. Or remove it altogether if it's not needed anymore.

Reply:
test_stopped() has a a nodiscard attribute, so dicarding the result will make the compiler show a warning.
`//` is a style decision on empty if blocks.